src/ui/components/Dashboard/DashboardLayout.jsx

```diff
@@ -1,49 +1,76 @@
 import React, { useState, useMemo, memo } from 'react';
+import axios from 'axios';
+import { useQuery } from 'react-query';
 import useStyles from './styles';
 import Sidebar from './Sidebar';
 import NavBar from './NavBar';
 import Collections from '../Collections';
 import Dashboard from './Dashboard';
+import Notifications from '../Notifications';
 
 function DashboardLayout() {
   const classes = useStyles();
   const [contentKey, setContentKey] = useState('dashboard');
+
+  const { data } = useQuery(['notifications'], () => axios.get(`/collection/notifications`));
+
+  const unviewedNotifs = useMemo(() => {
+    if (data) {
+      return data.data.filter(notif => {
```

Comment:
Since it's just one line in the function you can remove the `return` and curly braces
```suggestion
      return data.data.filter(notif => (!notif.viewed));
```